scheduler/pmc-processor-task/src/main/kotlin/ac/uk/ebi/pmc/data/MongoRepository.kt

```diff
@@ -19,36 +21,32 @@ class MongoRepository(
     private val dataBase: String,
     private val mongoClient: MongoClient
 ) {
-    fun getAllSubmissions(): FindIterable<SubmissionDoc> {
-        val database = mongoClient.getDatabase(dataBase)
-        val collection = database.getCollection(SUBMISSION_COLLECTION, SubmissionDoc::class.java)
+    fun getNotImportedSubmissionsBySourceFile(sourceFile: String): FindIterable<SubmissionDoc> =
```

Comment:
I think this method signature can be

```
findSubmission(source:String, Imported = false)
```

IMHO find plus kotlin named parameters make clear enought what does method do